etc/kubernetes/enterprise-gateway.yaml

```diff
@@ -142,6 +142,11 @@ spec:
 
         # Ensure the following VERSION tag is updated to the version of Enterprise Gateway you wish to run
         image: elyra/enterprise-gateway:dev
+        # resources:
+        #   limits:
+        #     cpu: "1"
+        #     memory: "2Gi"
+        #     nvidia.com/gpu: "0"
```

Comment:
The EG and KIP yaml settings have corresponding helm charts.  For those, it probably makes sense to parameterize the limits when helm is used.  The means of doing this is fairly straightforward if you'd like give this a try.

The EG definition is in the deployment.yaml file, while KIP is defined in daemonset.yaml and their respective "variables/values" would be listed in values.yaml.